shared/src/validation/eam-worker/eam-worker-create-backend/eam-worker-create.validation-schema.ts

```diff
@@ -0,0 +1,22 @@
+import * as Joi from 'joi';
+import { getNameOf } from '~/helpers/helpers';
+import { EAMWorkerCreateRequestDto } from '~/common/types/types';
+import {
+  EAMWorkerValidationMessageBackend,
+  EAMWorkerValidationRuleBackend,
+} from '~/common/enums/enums';
+
+const eamWorkerCreate = Joi.object({
+  [getNameOf<EAMWorkerCreateRequestDto>('password')]: Joi.string()
```

Comment:
you also need to validate `name`, `groupsIds` and etc (check `EAMWorkerCreateRequestDto` and validate all fields)